config/locales/en.yml

```
@@ -3254,7 +3256,7 @@ en:
       By default, OpenProject provides OAuth 2.0 authorization via %{authorization_code_flow_link}.
       You can optionally enable %{client_credentials_flow_link}, but you must provide a user on whose behalf requests will be performed.
     authorization_error: "An authorization error has occurred."
-    revoke_my_application_confirmation: "Do you really want to remove this application? This will revoke %{token_count} active for it."
+    disconnect_my_application_confirmation: "Do you really want to disconnect fropm this application? This will revoke %{token_count} active for it."
```

```suggestion
    disconnect_my_application_confirmation: "Do you really want to disconnect from this application? This will revoke %{token_count} active for it."
```

Also sentence felt strange by having active twice in it: `Do you really want to disconnect fropm this application? This will revoke one active token active for it.`, but I'm not sure if something can be done about it.